### DISCUSSION TITLE
Is there a published Roadmap, I couldn't anything in the existing Forums.

### DISCUSSION THREAD

**Discussion by nimblestart** [lowdefy/lowdefy#1263] [Questions and Help] [answered]
Is there a published Roadmap, I couldn't anything in the existing Forums.

**Comment by Gervwyk** [comment, marked as answer]
Hi @nimblestart 
Unfortunately currently not, we have been adding features as we require them for our production apps, and most of the dev effort is going into v4. 
We'll be sure to also publish an official roadmap as part of the docs update for v4.

**Reply by Gervwyk** [reply]
To expand on this, currently the focus for v4 for is on getting v4 stable with feature parity with v3, we're basically there, except for a few dev ex and cli ex things which needs attention. Then we'll do a big docs update to include all the new features, of which v4 plugins is the biggest game changer. 

In the future you can expect:
- More dev ex improvements (In progress)
- Dev tools (In progress)
- API endpoints
- VSCode plugin
- Static app server
- i18n
- Hosting service (In progress)
- Pub/Sub service
- Cron service
- Engine optimisations
- App analytics, audit and error logging
- More plugins, blocks, connections, operators and actions (In progress)
- More tutorials, examples and content.

@nimblestart anything in particular or ideas you where looking for?
